litmus-portal/backend/self-deployer/build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.14 AS builder
+
+ADD . /self-deployer
+WORKDIR /self-deployer
+RUN CGO_ENABLED=0 go build -o /output/deployer -v
+
+
+FROM alpine:latest
+
+RUN apk update && apk add curl bash
+RUN apk add --no-cache libc6-compat
```

Comment:
Do we need these commands?

Reply:
Yes these are also required for running some go compiled go libraries in alpine image

Reply:
curl and bash is not used in any part in thr program.

Reply:
Not sure about the lib6-compat

Reply:
Bash is required to run the executable properly. I had faced some problems without bash. Curl can be removed. I'll try without bash and check too.